900.version-fixes/x.yaml

```diff
@@ -107,7 +108,9 @@
 - { name: xrdp,                                                      ruleset: sisyphus,    untrusted: true } # Accused of fake 0.9.12.1
 - { name: xroach,                      verge: "12",                  ruleset: [opensuse,rpmsphere], incorrect: true }
 - { name: xrootconsole,                verlonger: 2,                 ruleset: void,        debianism: true }
-- { name: xsel,                        verpat: ".*20[0-9]{6}.*",                           snapshot: true }
+- { name: xsel,                        ver: "1.2.0",                                       outdated: true } # 1.2.0 is 12 years old
```

Comment:
Snapshots cannot be compared, and 1.2.0 is the latest official, so please leave as is. There also can't be `noscheme` on a subset of packages.

Reply:
see 3e039be